tools/cli/commands/create.py

```diff
@@ -94,6 +94,8 @@
 
 format_disk() {{
   echo "Formatting the persistent disk"
+  docker-credential-gcr configure-docker
+  docker pull {0}
```

Comment:
Are you doing the pull twice now? It looks like creategpu.py also still does the pull.

Reply:
I don't see any explicit pulls in creategpu.py, but it would be fine even if it does.

Running `docker pull` is idempotent; as long as the image hasn't changed it won't pull it down again.